src/GitObjectDb/Models/Rebase/ObjectRepositoryRebase.cs

```diff
@@ -21,58 +17,41 @@ namespace GitObjectDb.Models.Rebase
 {
     /// <inheritdoc />
     [ExcludeFromGuardForNull]
-    public class ObjectRepositoryRebase : IObjectRepositoryRebase
+    internal class ObjectRepositoryRebase : IObjectRepositoryRebase
     {
-        private readonly IServiceProvider _serviceProvider;
         private readonly IRepositoryProvider _repositoryProvider;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the field '_repositoryProvider' and declare it as a local variable in the relevant methods. ([csharpsquid:S1450](https://sonarcloud.io/organizations/frblondin-github/rules#rule_key=csharpsquid%3AS1450))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=GitObjectDb&pullRequest=33&issues=AWe4j_RLNP9b202rIM5M&open=AWe4j_RLNP9b202rIM5M)

[ISSUE_KEY]: AWe4j_RLNP9b202rIM5M